tools/pd-ctl/README.md

```diff
@@ -645,16 +644,16 @@ Usage:
 >> store label 1 zone cn               // Set the value of the label with the "zone" key to "cn" for the store with the store id of 1
 >> store weight 1 5 10                 // Set the leader weight to 5 and region weight to 10 for the store with the store id of 1
 >> store remove-tombstone              // Remove stores that are in tombstone state
->> store limit                         // Show limits of adding region operation for all stores
->> store limit region-add              // Show limits of adding region operation for all stores
->> store limit region-remove           // Show limits of removing region operation for all stores
+>> store limit                         // Show limits of adding peer operation for all stores
+>> store limit add-peer                // Show limits of adding peer operation for all stores
+>> store limit remove-peer             // Show limits of removing peer operation for all stores
 >> store limit all 5                   // Limit 5 adding region operations per minute for all stores
```

Comment:
```
store limit all 5
```
the command only set for `add-peer`,  can we set `remove-peer` at the same time?

Reply:
I'm not sure it's better. What is your opinion? @lhy1024

Reply:
Before, the user only needed one command. Now users think that the speed has been released, but in fact there is still a speed limit for `remove-peer`.